### PR TITLE
[Teams] WARP docs + tunnel fixes

### PR DESCRIPTION
**Pull request by abracchi-tw**

### CODE REVIEW

**Review by kkrum — APPROVED** [review]

File: products/cloudflare-one/src/content/connections/connect-apps/do-more-with-tunnels/secure-server.md [hunk 1]
```diff
@@ -4,19 +4,18 @@ order: 6
...
 
-| Before you start |
-|---|
-| 1. [Create a Tunnel](/connections/connect-apps/create-tunnel) |
-| 2. [Configure the Tunnel](/connections/connect-apps/configuration) |
-| 3. [Configure routing to the Tunnel](/connections/connect-apps/routing-to-tunnel) |
-| 4. [Run the Tunnel](/connections/connect-apps/run-tunnel) |
+<Aside type='note' header='Before you start'>
+
+Make sure you follow our [guide](/connections/connect-apps/install-and-setup/tunnel-guide) to create, configure, and run a tunnel.
```
Comment (nayaabahsan) [inline products/cloudflare-one/src/content/connections/connect-apps/do-more-with-tunnels/secure-server.md:9]: The paragraph below capitalizes "a Tunnel," so it should probably be consistent here. Not sure if this instance should be capitalized or the below instance should be made lower case.

File: products/cloudflare-one/src/content/connections/connect-devices/warp/exclude-traffic/index.md [hunk 2]
```diff
@@ -3,11 +3,11 @@ order: 14
 pcx-content-type: concept
 ---
  
-# Exclude network traffic from WARP
+# Exclude or include network traffic with WARP
 
-When the WARP client is deployed, all DNS requests and/or network traffic on the device are processed by Cloudflare Gateway by default. However, under certain circumstances, you may need to exclude DNS requests and/or network traffic from being processed by Gateway.
- 
-To do that, there are two settings you can use depending on your needs:
+When the WARP client is deployed, all DNS requests and/or network traffic on the device are processed by Cloudflare by default. However, under certain circumstances, you may need to exclude DNS requests and/or explicitly exclude or include network traffic depending on your needs.
```
Comment (nayaabahsan) [inline products/cloudflare-one/src/content/connections/connect-devices/warp/exclude-traffic/index.md]: It reads to me like "under certain circumstances" and "depending on your needs" in this last sentence are conveying they same thing, and one could be taken out? so either:
"However, under certain circumstances, you may need to exclude DNS requests and/or explicitly exclude or include network traffic."
or
"However, you may need to exclude DNS requests and/or explicitly exclude or include network traffic depending on your needs."

**Review by nayaabahsan — APPROVED** [review]

File: products/cloudflare-one/src/content/connections/connect-apps/do-more-with-tunnels/secure-server.md (hunk 1 above)
Comment (abracchi-tw) [reply products/cloudflare-one/src/content/connections/connect-apps/do-more-with-tunnels/secure-server.md:9]: great catch!

File: products/cloudflare-one/src/content/connections/connect-devices/warp/exclude-traffic/index.md (hunk 2 above)
Comment (abracchi-tw) [reply products/cloudflare-one/src/content/connections/connect-devices/warp/exclude-traffic/index.md]: fixed!

**Pull request merged**